queries/typescript/highlights.scm

```diff
@@ -55,11 +58,21 @@
 (pair
   ":" @punctuation.delimiter)
 
+(index_signature
+  ":" @punctuation.delimiter)
+
+(opting_type_annotation
+  "?:" @punctuation.delimiter)
```

Comment:
is this not an operator?

Reply:
No, it is actually the same notation as `function (optionalParam?: string)`.
I tried to make the `?` sigil a `punctuation.special`, but the grammar interprets `?:` as one symbol.